spec/features/search/index_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 
 describe "Search index page" do
-  it "can enter a name and show all obituaries with that name" do
+  it "can enter a name and show all obituaries with that name", :vcr do
```

Comment:
This may cause a conflict with the tests I am setting up to work with Javascript, but I can try to figure that out on my end when this is merged.

Reply:
Good thinking Jack!